tests/integration/local/test_headers_include.py

```diff
@@ -0,0 +1,28 @@
+import unittest, PyRDF, ROOT
+
+class IncludesLocalTest(unittest.TestCase):
+    """
+    An integration test to check that the required
+    header files are properly included.
+
+    """
+    def test_includes_function_with_filter_op(self):
+        """
+        An integration test to check that the filter
+        operation is able to use C++ functions that
+        were include using header files.
+
+        """
+        PyRDF.include("tests/integration/local/test_headers/header1.hxx")
+
+        rdf = PyRDF.RDataFrame(10)
+        rdf_defined = rdf.Define("b1", "(double) tdfentry_")\
+                         .Define("b2", "(int) tdfentry_ * tdfentry_")
+
+        # This filters out all numbers less than 40 in column 'b2'
+        rdf_filtered = rdf_defined.Filter("check_number_less_than_40(b2)")
```

Comment:
To make this test simpler, what about just `rdf_defined.Filter("check_number_less_than_5(tdfentry_)")`, and you do not need the previous Defines.

Reply:
Yup @etejedor . That is better. Will make this fix 👍